bdk-python/tests/test_live_wallet.py

```diff
@@ -23,7 +23,16 @@ def test_synced_balance(self):
 
         self.assertGreater(wallet.get_balance().total, 0)
 
-    def test_broadcast_transaction(self):
+        print(f"Transactions count: {len(wallet.transactions())}")
+        transactions = wallet.transactions()[:3]
+        for tx in transactions:
+            sent_and_received = wallet.sent_and_received(tx)
+            print(f"Transaction: {tx.txid()}")
+            print(f"Sent {sent_and_received.sent}")
+            print(f"Received {sent_and_received.received}")
+
+
+def test_broadcast_transaction(self):
```

Comment:
Small nit: your indentation here should be 4 spaces to the right so that the def is inside the class definition.

Reply:
good call, updated now ✅